oecologia-australis.csl

```diff
@@ -260,10 +259,15 @@
   </macro>
   <macro name="publisher">
     <choose>
-      <if type="report" match="any">
-        <group delimiter=": ">
-          <text variable="publisher-place"/>
-          <text variable="publisher"/>
+      <if type="report" match="all">
+        <group delimiter=" " prefix=" ">
+          <choose>
+            <if match="any" variable="number-of-pages publisher">
+              <text variable="number-of-pages" prefix="p. " suffix="."/>
+              <text variable="publisher" prefix="ed. " suffix=", "/>
```

Comment:
`<text variable="publisher" prefix="ed. " suffix=", "/>` is also rather odd, because "ed." usually stands for "editor", not "publisher".